### PR TITLE
add module website_slides_qq_video

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: website_slides_qq_video/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Website Slides upload QQ video
+==============================
+This model  extends the functionality of and allows to publish the QQ video
```
Comment (siyuancn) [inline website_slides_qq_video/README.rst]: the functionality of website_slides

**Comment by elicoidal** [comment]
to be published in OCA/l10n-china
website_slides in OCA/website repo

File: website_slides_qq_video/README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+in the channel
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (website_slides)
+
+Bug Tracker
```
Comment (elicoidal) [inline website_slides_qq_video/README.rst:40]: @siyuancn I need a usage section explaining how to upload the qq videos and the format of the URL for qq.

**Comment by siyuancn** [comment]
@Rona111 Url format is : <address>?vid=<vid>,
The vid is the key for video.

You can get it when you click any qq video most of the time, etc:
http://v.qq.com/cover/k/kg0cy3syh1ovu9z.html?vid=a001986fc2v,

But if you can't find the vid in the url, like:
http://v.qq.com/boke/page/n/0/o/n0163mjor1o.html

Click the share button under the video and find the iframe code for the video like:
<iframe frameborder="0" width="640" height="498" src="http://v.qq.com/iframe/player.html?vid=n0163mjor1o&tiny=0&auto=0" allowfullscreen></iframe>

Just copy and paster the url in the src: http://v.qq.com/iframe/player.html?vid=n0163mjor1o

**Comment by noahzaozao** [comment]
LGTM

**Pull request merged**